### ISSUE TITLE
Kit is using the wrong php version

### ISSUE THREAD

**Issue by SamuelNitsche** [tmdh/laravel-kit#15]
When I open a project, I get the following error message.

```
Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 7.4.0". You are running 7.3.11. in /xxx/vendor/composer/platform_check.php on line 24
```

Running `php -v` outputs the correct PHP version which I am using.
```
PHP 8.0.0 (cli) (built: Nov 30 2020 13:47:29) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.0-dev, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.0, Copyright (c), by Zend Technologies
```

Kit version: 2.0.0
Operating system: macOS 10.15.7

**Comment by ejunker** [comment]
I also got this same error message. I am running version 2.0.0 on macOS.

**Comment by tmdh** [comment]
There may be multiple `php` installations in your system. Remove old ones if you can or see if there is any environment variable `path` precedence issues.
I'll add the ability to choose custom PHP executable in the next release.

**Comment by crossworth** [comment]
I think OSX comes with PHP bundled, I dont think is a `path variable` issue, otherwise the terminal would be returning the wrong version as well.

PHP installed with homebrew is at (**version which I use**):  
- `/usr/local/bin/php`

PHP bundled is at:
- `/usr/bin/php`

Content of `/etc/paths`:
```sh
❯ cat /etc/paths
/usr/local/bin
/usr/bin
/bin
/usr/sbin
/sbin
```

Changing the path variable or `/etc/paths` file don't work.

The path that Laravel Kit uses:
![image](https://user-images.githubusercontent.com/1251151/109524021-c1c83400-7a8e-11eb-91a5-9e7b481473e5.png)

My path:
![image](https://user-images.githubusercontent.com/1251151/109524210-ffc55800-7a8e-11eb-98e4-24cacf6e8426.png)


Maybe is related to this issue: [https://stackoverflow.com/questions/62067127/path-variables-empty-in-electron](https://stackoverflow.com/questions/62067127/path-variables-empty-in-electron)

**Comment by tmdh** [comment]
I think this should do the work.
```js
if (process.platform === "darwin") {
  process.env.PATH = ["/usr/local/bin", process.env.PATH].join(":");
}
```
Unfortunately, I can't test this because I don't have a mac.

**Comment by SamuelNitsche** [comment]
Not really. I have a m1 mac and php is located somewhere else. I think using this package makes more sense.

https://www.npmjs.com/package/fix-path

**Comment by tmdh** [comment]
This package has a webpack bug saying it can't resolve a module. The dependencies are outdated. I need to fork this and fix the bugs.

**Comment by SamuelNitsche** [comment]
I just built the app from source using the latest code and now it works fine for me.

**Comment by tmdh** [comment]
Fixed in v2.0.1
